kubernetes/samples/scripts/domain-lifecycle/stopServer.sh

```diff
@@ -0,0 +1,120 @@
+# !/bin/sh
```

Comment:
General comment:  Sorry for not noticing this earlier, but if I understand the code correctly, none of the scripts detect when a start policy _already_ matches the target policy.  In that case, the scripts should exit early without doing anything and should print a helpful info message to let the user know. (And they definitely  shouldn't adjust replica count).

Reply:
Good point. I have added this validation for cluster and domain start/stop scripts. The server start/stop scripts already checks to see if server is already started/stopped. Thanks.